firewalldb/kvstores_test.go

```diff
@@ -130,21 +135,15 @@ func testTempAndPermStores(t *testing.T, featureSpecificStore bool) {
 	require.True(t, bytes.Equal(v1, []byte{4, 3, 2}))
 	require.True(t, bytes.Equal(v2, []byte{6, 5, 4}))
 
-	// Close the db.
-	require.NoError(t, db.Close())
+	// Re-init the DB.
```

Comment:
we could maybe call `db.Stop()` before, even though it's a no-op, also at the end of the test

Reply:
added 👍